app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -1,19 +1,42 @@
+const META_DATA_MAX_LENGTH = 50
+const MAX_NO_OF_METADATA_COLUMNS = 10
```

Comment:
It would be good for these values to come from JS commons to have parity with the definitions in https://github.com/alphagov/pay-java-commons/blob/master/model/src/main/java/uk/gov/pay/commons/model/charge/ExternalMetadata.java. 

This could be done as a separate PR, I'd be happy to do this and it can be updated after, it shouldn't block this

Reply:
I have updated this PR with hardcoded values.

I will make an update to pay-js-commons to include these values and then update Self service.